warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java

```diff
@@ -334,10 +334,14 @@ public CloseableIterable<QueryData> process(GenericQueryConfiguration genericCon
             configCopy.setBeginDate(dateRange.getLeft());
             configCopy.setEndDate(dateRange.getRight());
 
+            // we want to make sure the same query and same query id are used for tracking purposes and execution
+            configCopy.setQuery(originalConfig.getQuery());
```

Comment:
Is it okay to share the same object across two instances of the planner? I would think we should copy the Query object but maybe that's an incorrect assumption on my part.

Reply:
The settings object is static at this point, so no need to have a copy

Reply:
Perfect

Reply:
But that being said, perhaps safety.... looking at it.

Reply:
looks like the query object is actually already duplicated, so all I need to do is reset the id..... testing